Story/Friday13th/CoreFriday13th.cs

```diff
@@ -0,0 +1,825 @@
+        
+        // Take Sleezter Bunny Out! 1968
+        Story.KillQuest(1968, "Sleezter", "Sleezter Bunny");
+        
+        // Mantis Ray Repair 1969
+        Story.MapItemQuest(1969, "Sleezter", 976, 6);
+        
+        // A Map Wouldn't Hurt 1970
+        Story.KillQuest(1970, "Sleezter", "BunnyMinion");
+        
+        // Endin' Minions 1971
+        Story.MapItemQuest(1971, "Sleezter", 977);
+        
+        // Breaking Bad Eggs 1972
+        Story.MapItemQuest(1972, "Sleezter", 978);
+
+        // Mantis Ray
+        Story.ChainQuest(2021);
+    }
+
+    public void Neverhub()
+    {
+        // Check progress and membership
+        if (Core.isCompletedBefore(2234) || !Core.IsMember && !CalculateFriday13())
+            return;
+
+        // Preload quests
+        Story.PreLoad(this);
+
+        // Find the Shadow's Door 2222
+        Story.MapItemQuest(2222, "Neverworld", 1315);
+
+        // One-way Trip to Neverworld 2223
+        Story.KillQuest(2223, "Neverworld", "Whablobble");
+
+        // Slash the Shadows 2224
+        Story.KillQuest(2224, "Neverworld", "Spid-Squider");
+
+        // Deprogramming for Dummies 2225
+        Story.MapItemQuest(2225, "Neverworld", 1316, 10);
+        Story.KillQuest(2225, "Neverworld", "Snackistopheles");
+
+        // Guardian of the Lab-rary 2226
+        Story.MapItemQuest(2226, "Neverworld", 1317);
+        Story.KillQuest(2226, "Neverworld", new[] { "Snackistopheles", "Spid-Squider", "Snackistopheles", "Spid-Squider", "Fishizzle" });
+
+        // Quick switch! 2227
+        Story.MapItemQuest(2227, "Neverworld", 1318, 5);
+        Story.MapItemQuest(2227, "Neverworld", 1326);
+
+        // We're BOOM-ed! 2228
+        Story.MapItemQuest(2228, "Neverworld", 1321);
+
+        // De-Generation Situation 2229
+        Story.KillQuest(2229, "Neverworld", "Generator");
+
+        // Shielded Against the Shadows 2230
+        Story.MapItemQuest(2230, "Neverworld", 1319);
+        Story.KillQuest(2230, "Neverworld", "Fishizzle");
+
+        // Making a Many-armed Army 2231
+        Story.KillQuest(2231, "Neverworld", "Kennel Door");
+
+        // Kreature Kibble  Khaos 2232
+        Story.MapItemQuest(2232, "Neverworld", 1320, 10);
+
+        // The Gang's Not All Here 2233
+        Story.KillQuest(2233, "Neverworld", "Fishizzle");
+
+        // Beast Maker Beat-down 2234
+        Story.ChainQuest(2234);
+    }
+
+    public void Battledoom()
+    {
+        // Check membership
+        if (Core.IsMember && CalculateFriday13())
+            return;
+     
+        // Mark items to not bank
+        Core.BankingBlackList.AddRange(new[] { "Unlucky Gem I", "Unlucky Gem II", "Unlucky Gem III",
+                                               "Unlucky Gem IV", "Unlucky Gem V", "Unlucky Gem VI",
+                                               "Unlucky Gem VII", "Cursed Mirror of Enutrof", "Shadowglass Shard" });
+
+        // Check if item is in inventory
```

Comment:
same here no need for comments to describe the functionality of the line of code